### DISCUSSION TITLE
Do you feel the need to better integrate your work schedule with Freeplane?

### DISCUSSION THREAD

**Discussion by bepolymathe** [freeplane/freeplane#1014] [Polls]
Hi all, 

When managing your notes, documentation and tasks in Freeplane you may feel the need for a better integration of a calendar. Is this feeling shared by other users? 

PS: Don't you think that opinion polls are not used very much?

**Comment by ldmpub** [comment]
Explanation on my vote: there are already reminders in Freeplane ... and I don't use them.

If this is a personal initiative and you have the required dev skills ... just go on 👏. 

If your idea is to ask Freeplane community where to put its effort, I will personally give my voice to other initiatives like: New nodes layouts or building an Android app.

**Reply by bepolymathe** [reply]
> If this is a personal initiative and you have the required dev skills ... just go on .

Unfortunately not. 




> If your idea is to ask Freeplane community where to put its effort, I will personally give my voice to other initiatives like: New nodes layouts or building an Android app.

It makes complete sense. I especially wanted to see if this would be of interest to some (or even already experimented with). But I agree with you, there are more urgent priorities (android app). 

PS: I must admit that I also wanted to try the opinion polls ;-)

**Comment by theelectricsoul** [comment]
I know there are more urgent matters....  but caldav would be awesome.....   :)   the cloud to rule them all :)

**Comment by sgjohnson1981** [comment]
I actually went in search of something that would do this. I found Redmine, which has a mindmap plugin. Unfortunately it's a entire stack, and you need to be fairly adept at server maintenance to use it because the documentation isn't that great. Such is life with open-source software.
